ideas/new-ops.txt

```diff
@@ -0,0 +1,117 @@
+# Proposal for new stdlib operations
+
+This is a proposal for several APIs that could be added to stdlib.
+
+## `builtin` candidates
+
+Entity	Signature	Notes
+
+`Not`	`Not(x bool) (x bool)`	Return logical not operation applied to input `x`
+`Pair<A, B>`	`struct { first A; second B }`	A generic pair of items. Can stand in for other pair types we've introduced such as `ZipResult`
```

Comment:
1. Do you feel like we gonna need this a lot?
2. If yes, - enough to be inside builtin?